build.sbt

```diff
@@ -112,6 +112,17 @@ lazy val tapirCats: Project = (project in file("cats"))
   )
   .dependsOn(core)
 
+lazy val tapirRefined: Project = (project in file("refined"))
```

Comment:
can you put it in `integrations/refined`? I'll move `cats` there later as well

Reply:
I did it for cats too!

Reply:
Thanks!